userguide/glossary/Kill-Bill-Glossary.adoc

```diff
@@ -92,6 +103,10 @@ _Example:_
 
 |===
 
+External Payment::  A built-in payment method that can be used to track payments that occur outside of Kill Bill.
+
+Event:: A message exposed by Kill Bill core components that triggers certain system actions. For example, when a subscription is created, Kill Bill triggers an event that leads to invoice generation. 
```

Comment:
I used the Events guide to come up with: "A type of object that Kill Bill creates when a certain occurrence takes place in the system. The system can respond to these events with various actions, such as when a subscription is created, Kill Bill generates an invoice."

Reply:
An event is created as a result of something happening in the system. Example: Creating a subscription will generate an event notifying listeners about this new state. I would remove the fact that it `triggers certain system actions`, as this is an internal implementation choice that we have made for the system.

Reply:
I've used a combination of both your comments, LMK how this looks.